driftbase/api/matches.py

```diff
@@ -586,7 +586,7 @@ def put(self, args, match_id):
 
                 if new_status == "started":
                     match.start_date = utcnow()
-                elif new_status == "completed":
+                elif new_status in ("completed", "ended"):
```

Comment:
I see "end" in match active checks but is missing here.

Fixing battle server so aborted servers will have match_status set to "ended".